src/utils/analysis-utils.ts

```diff
@@ -434,8 +435,31 @@ export async function loadFeaturesClientSide(
 }
 
 export function getAnalysisTableColumns(
-  analysisResult: BaselineLayerResult,
+  analysisResult: BaselineLayerResult | PolygonAnalysisResult,
 ): Column[] {
+  if ('tableColumns' in analysisResult) {
+    return analysisResult.tableColumns.map(column => {
+      return {
+        id: column as any,
+        // remove prefix from column labels
```

Comment:
This should be handled by functions specific to zonal

Reply:
if I understand correctly, I should create a new function called something like `getPolygonAnalysisTableColumns`?

Reply:
No, I think this could simply be moved inside `requestAndStorePolygonAnalysis` for now

Reply:
ah. okay. will do.

Reply:
done